azure-monitor-opentelemetry/tests/configuration/test_configure.py

```diff
@@ -458,37 +340,7 @@ def test_setup_instrumentations_lib_not_supported(
         ep2_mock.name = _SUPPORTED_INSTRUMENTED_LIBRARIES[1]
         ep2_mock.load.return_value = instr_class_mock
         dep_mock.return_value = None
-        _setup_instrumentations(configurations)
-        dep_mock.assert_called_with(ep2_mock.dist)
-        ep_mock.load.assert_not_called()
-        ep2_mock.load.assert_called_once()
-        instrumentor_mock.instrument.assert_called_once()
```

Comment:
This diff turned out weird, but if you expand below, you'll see that I mainly just removed test_setup_instrumentations_lib_excluded.